Framework/Core/Windows/Devices/Pwm/PwmController.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Windows.Devices.Pwm.Provider;
+
+namespace Windows.Devices.Pwm
+{
+    public sealed class PwmController
+    {
+        private IPwmControllerProvider m_provider;
+        private double m_minFrequency;
+        private double m_maxFrequency;

```

Comment:
Providers should report their greatest range (lowest min and highest max).  This API will guard against users setting anything outside that range.  However, more complex behavior, such as variable sub-ranges must be arbitrated by the Provider.

This is the philosophy adopted by the designers of these APIs.  We're trying to mirror those decisions as closely as possible for the sake of cross-device consistency.